src/game.cpp

```diff
@@ -9078,29 +9078,82 @@ bool game::check_safe_mode_allowed( bool repeat_safe_mode_warnings )
         return true;
     }
     // Monsters around and we don't want to run
-    std::string spotted_creature_name;
+    std::string spotted_creature_text;
     const monster_visible_info &mon_visible = u.get_mon_visible();
     const auto &new_seen_mon = mon_visible.new_seen_mon;
 
+    const nc_color mon_color = c_red;
+    const nc_color dir_color = c_light_blue;
     if( new_seen_mon.empty() ) {
         // naming consistent with code in game::mon_info
-        spotted_creature_name = _( "a survivor" );
+        spotted_creature_text = colorize( _( "a survivor" ), mon_color );
         get_safemode().lastmon_whitelist = get_safemode().npc_type_name();
+    } else if( new_seen_mon.size() == 1 ) {
+        const auto &mon = new_seen_mon.back();
+        const std::string dir_text = string_format( _( "to the %s" ),
```

Comment:
Please add translator comments to strings like these, here and below.

Reply:
Could you point me at an example?

Reply:
`//~ %1$s: book name, %2$s: skill name, %3$d and %4$d: skill levels`

Something like this.